content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -0,0 +1,228 @@
++++
+title = "Kubeflow Pipelines Local Cluster Deployment"
+description = "Instructions to deploy Kubeflow Pipelines standalone to a local cluster for testing purposes"
+weight = 20
++++
+
+As an alternative to deploying Kubeflow Pipelines (KFP) as part of the
```

Comment:
Refactoring here and everywhere else for more clarity. Changed the syntax and other things. 

What do you think?

I've done my own research on kind, k3s and k3s on WSL on their respective sites to help you with the PR.

```diff
- As an alternative to deploying Kubeflow Pipelines (KFP) as part of the
- [Kubeflow deployment](/docs/started/getting-started/#installing-kubeflow), you also have a choice
- to deploy only Kubeflow Pipelines. Follow the instructions below to deploy
- Kubeflow Pipelines as part of your local environment for test purposes using the supplied kustomize manifests.
...
- This guide currently describes how to install Kubeflow Pipelines standalone
-  on Kind,K3s and K3s on WSL2 other configurations will be added here covering - minikube(TBD) and others.

+ This guide shows how to deploy only Kubeflow Pipelines—Kubeflow Pipelines Standalone using:
+ 
+ - kind
+ - k3s
+ - k3s on WSL

+ It can be part of your local environment using the supplied kustomize manifests for test 
+ purposes. The guide is an alternative to [deploying Kubeflow Pipelines (KFP)](/docs/started/getting-started/#installing-kubeflow).
```

Reply:
Feel free to mix and match.

Reply:
It's "k3s on WSL (not WSL2)", since you upgrade to WSL2 after getting WSL. If this is incorrect, change it everywhere, in case I missed it.